code/__HELPERS/honkerblast.dm

```diff
@@ -34,14 +34,14 @@
 	for(var/mob/living/carbon/victim in properly_honked)
 		victim.Paralyze(20)
 		victim.Stun(50)
-		victim.Jitter(250)
+		victim.set_timed_status_effect(500 SECONDS, /datum/status_effect/jitter, only_if_higher = TRUE)
 		var/obj/item/organ/ears/ears = victim.getorganslot(ORGAN_SLOT_EARS)
 		ears?.adjustEarDamage(7, 10)
 		to_chat(victim, "<font color='red' size='5'>HONK</font>")
 
 	for(var/mob/living/carbon/victim in lightly_honked)
 		victim.Knockdown(20)
-		victim.Jitter(100)
+		victim.set_timed_status_effect(100 SECONDS, /datum/status_effect/jitter, only_if_higher = TRUE)
```

Comment:
for every other setter you double the original value, why is this one the same?

Reply:
User error